pywatts/wrapper/sm_time_series_model_wrapper.py

```diff
@@ -0,0 +1,170 @@
+import inspect
+from typing import Dict, Type
+
+import numpy as np
+import xarray as xr
+from statsmodels.iolib import load_pickle
+from statsmodels.tsa.base.tsa_model import TimeSeriesModel
+
+from pywatts.core.filemanager import FileManager
+from pywatts.utils._xarray_time_series_utils import _get_time_indeces, numpy_to_xarray
+from pywatts.wrapper.base_wrapper import BaseWrapper
+
+
+class SmTimeSeriesModelWrapper(BaseWrapper):
```

Comment:
Quite complicated class name. Is it may possible to have an StatsModelWrapper allowing all statsmodel models? Or naming it different? StatsModelsTSAWrapper? Just an idea.

Reply:
I wanted to avoid TSA, since I suppose that we cannot wrap all TSA models. However we can wrap all models that inherit from  tsa_model.TimeSeriesModel